toolchain/sem_ir/formatter.cpp

```diff
@@ -769,11 +769,11 @@ class Formatter {
     out_ << InstT::Kind.ir_name();
     FormatInstructionRHS(inst);
     if (auto const_id = sem_ir_.constant_values().Get(inst_id);
-        const_id.is_valid()) {
-      out_ << ", const";
-      if (const_id != inst_id) {
+        const_id.is_constant()) {
+      out_ << (const_id.is_symbolic() ? ", sym" : ", const");
```

Comment:
Similar to the below, what do you think about two formatting changes:

1) Render these differently from arguments, as some kind of annotation? My best guess is `[...]`, but happy for other "annotation"-y formats.

2) Spell out the kinds, and use the actual names?

```suggestion
      out_ << (const_id.is_symbolic() ? " [symbolic" : " [template");
```